src/components/core/DpBulkEditHeader.vue

```diff
@@ -5,39 +5,32 @@
     <span class="u-mr">
       {{ selectedItemsText }}
     </span>
-    <slot :selected-items-count="selectedItemsCount">
-      <!-- place edit actions here -->
-    </slot>
+    <slot />
     <button
       type="button"
       class="btn btn--secondary float-right"
       @click="$emit('reset-selection')">
-      {{ Translator.trans('unselect') }}
+      {{ deselect }}
     </button>
   </div>
 </template>
 
 <script>
+import { de } from '~/components/shared/translations'
+
 export default {
   name: 'DpBulkEditHeader',
 
   props: {
-    selectedItemsCount: {
-      type: Number,
-      required: false,
-      default: 0
-    },
-
-    selectionText: {
+    selectedItemsText: {
       type: String,
-      required: false,
-      default: ''
+      required: true
     }
   },
 
   computed: {
-    selectedItemsText () {
-      return this.selectedItemsCount + ' ' + this.selectionText
+    deselect () {
+      return de.operations.deselect.all
```

Comment:
this don't have to be a computed. It should be `data`. Or maybe a prop with a default value.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/615/commits/4624c5b63418c5d63cc6507befd195f69f52516f

Reply:
Why did You not move it to `props` ? Did You thought about it?

Reply:
imho this **must not** be a prop, otherwise we would have to implement props for every translation key within demosplan-ui. i thought the plan is to solve that on a library wide level, maybe when adding vue-i18n.